mail_show_follower/models/res_company.py

```diff
@@ -5,5 +5,25 @@ class ResCompany(models.Model):
     _inherit = "res.company"
 
     show_internal_users_cc = fields.Boolean(
-        string="Show Internal Users CC", default=True
+        string="Show Internal Users CC",
+        default=True,
+    )
+    show_followers_message_sent_to = fields.Html(
+        string="Text 'Sent to'",
+        translate=True,
+        default="This message has been sent to",
+    )
+    show_followers_partner_format = fields.Char(
+        string="Partner format",
+        default="%(partner_name)s",
+        help="Supported parameters:\n"
+        "%(partner_name)s = Partner Name\n"
+        "%(partner_email)s = Partner Email\n"
+        "%(partner_email_domain)s = Partner Email Domain\n"
+        "Format on plain HTML: &lt;(<), &gt;(>)",
```

Comment:
It's better to not use plain HTML as format. Instead, do automatic escaping when interpolating the values. IMHO it provides a better UX because asking an average user to write HTML escape sequences is too hard, and opens a venue for unexpected behaviours.

This point is important.

You can just wrap the template with [`markupsafe.escape`](https://markupsafe.palletsprojects.com/en/2.1.x/escaping/#markupsafe.escape) like Odoo does, when using it (see other comment).

Even better, instead of asking the user to write a %(template)s, a nice UX enhancement would be to use a Selection field instead, so you just give the user 3 options:

1. Mitchell Admin
2. Mitchell Admin (admin@example.com)
3. Mitchell Admin (example.com)

The simpler, the better! 😃